packages/cardano-services/test/Reward/RewardsHttpService.test.ts

```diff
@@ -174,12 +174,15 @@ describe('RewardsHttpService', () => {
           expect(response).toMatchSnapshot();
         });
         it('returns address rewards history with epochs ', async () => {
+          const rewardAccount = Cardano.RewardAccount(
+            'stake_test1up32f2hrv5ytqk8ad6e4apss5zrrjjlrkjhrksypn5g08fqrqf9gr'
+          );
           const response = await provider.rewardsHistory({
             epochs: {
-              lowerBound: 10,
-              upperBound: 11
+              lowerBound: 75,
+              upperBound: 76
             },
-            rewardAccounts: [rewardAcc]
```

Comment:
Looks good, but would be great to rename at least one of `rewardAcc` or `rewardAccount` to communicate the intent/difference between the variables

Reply:
What's the issue?